tests/filter.py

```diff
@@ -132,6 +134,32 @@ def filter_saturation(_):
     return _base_scene(ngl.FilterSaturation(1.5))
 
 
+@test_cuepoints(points=_CUEPOINTS, keyframes=1, tolerance=1)
+@ngl.scene()
+def filter_selector_light(_):
+    return _base_scene(
+        ngl.FilterSelector(
+            range=(0.41, 0.5),
+            component="lightness",
+            drop_mode="inside",
+            output_mode="colorholes",
+        )
+    )
+
+
+@test_cuepoints(points=_CUEPOINTS, keyframes=1, tolerance=1)
+@ngl.scene()
+def filter_selector_hue(_):
+    return _base_scene(
+        ngl.FilterSelector(
+            range=(-0.1 * math.tau, 0.45 * math.tau),
+            component="hue",
```

Comment:
Can you also exercise the "chroma" component ?

Reply:
added